lib/parsers.js

```diff
@@ -94,7 +93,7 @@ module.exports = function(nanomatch, options) {
       if (!m) return;
       var val = m[0];
 
-      var isNegated = isOdd(val.length);
+      var isNegated = Math.abs(val.length%2)==1;
```

Comment:
This change is great, but I doubt you need to worry about strings with negative length :)

Reply:
Yeah just being careful. I can take out the Math.abs if you want :)

Reply:
It seems like being careful is how we got to `isOdd` in the first place.